extensions/processing/json-schema/schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.stacspec.org/v1.0.0-beta.2/extensions/proc/json-schema/schema.json#",
+  "title": "Processing Extension",
+  "description": "STAC Processing Extension to a STAC Item or STAC Collections",
+  "allOf": [
+    {
+      "$ref": "../../../item-spec/json-schema/item.json"
+    },
+    {
+      "$ref": "#/definitions/proc"
+    },
+    {
+      "allOf": [
+        {
+          "$ref": "../../../collection-spec/json-schema/collection.json"
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
+        },
+        {
+          "$ref": "#/definitions/scientific"
+        }
+      ]
+    }
+  ],
+  "definitions": {
+    "proc": {
+      "type": "object",
+      "required": [
+        "stac_extensions",
+        "properties"
+      ],
+      "properties": {
+        "stac_extensions": {
+          "type": "array",
+          "contains": {
+            "enum": [
+              "proc",
+              "https://schemas.stacspec.org/v1.0.0-beta.2/extensions/proc/json-schema/schema.json"
+            ]
+          }
+        },
+        "properties": {
+          "type": "object",
+          "required": [
+            "proc:level"
```

Comment:
Should proc:level be required or just any of the fields (i.e. `minProperties: 1`)?